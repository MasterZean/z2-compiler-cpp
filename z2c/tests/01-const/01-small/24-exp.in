val a: Small = -0s
val b: Small = 0s
val c: Small = -1s
val d: Small = 1s
val e: Small = -100s
val f: Small = 100s
val g: Small = -127s
val h: Small = 127s
val i: Small = -128s
val j: Small = 128s
val k: Small = -200s
val l: Small = 200s

