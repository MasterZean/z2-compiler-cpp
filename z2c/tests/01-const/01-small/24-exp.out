int8 a = 0;
int8 b = 0;
int8 c = -1;
int8 d = 1;
int8 e = -100;
int8 f = 100;
int8 g = -127;
int8 h = 127;
int8 i = -128;
DummyClass(10, 14): error:
	can't assign 'Int' instance to 'Small' instance
DummyClass(11, 14): error:
	can't assign 'Int' instance to 'Small' instance
DummyClass(12, 14): error:
	can't assign 'Int' instance to 'Small' instance
